src/jenkinsgithublander/jobs.py

```diff
@@ -104,6 +119,7 @@ def mark_pull_request_build_failed(job_name, pr, build_number, failure_message,
         config['jenkins.merge.url'],
```

Comment:
so this whole jenkins info isn't required? Can we just remove the whole bit vs adding the None to the end?